test/resources/GeneratedTests.spec.js

```diff
@@ -768,8 +768,8 @@ describe('TaxId', function() {
 
   it('retrieveTaxId method', async function() {
     const taxId = await stripe.customers.retrieveTaxId(
-      'txi_xxxxxxxxxxxxx',
-      'cus_xxxxxxxxxxxxx'
+      'cus_xxxxxxxxxxxxx',
+      'txi_xxxxxxxxxxxxx'
```

Comment:
Hmmmmm....... maybe we have something non-deterministic here? 🤔  Seems unrelated to this...

Reply:
Not non-determinism -- this is the result of a minor bugfix: the correct ordering is [parentId, childId] and this had been called with the arguments in the wrong order.